scss/_type.scss

```diff
@@ -244,19 +79,15 @@ h2,
 //
 
 // Builds on `abbr`
-.initialism {
-  @include font-size($initialism-font-size);
-  text-transform: uppercase;
-}
+// OUDS mod: no .initialism
```

Comment:
Just thought back to it, thanks for raising this issue. 🙏 

My reflexion: 
- `.initialism` should only be used on top of `<abbr>` on Bootstrap side
- On Orange side, we don't want `.initialism` to be used 
- If we keep it, we'll need to add a callout saying that even with Bootstrap compatibility `on`, it shouldn't be used
- If we remove it, people using it will only default to `<abbr>` style which is the wanted thing from an Orange pov

I don't see any drawback removing it right now, if you see any, please let me know.